cdserver/db/migrate/20170327144759_create_units.rb

```diff
@@ -0,0 +1,14 @@
+class CreateUnits < ActiveRecord::Migration[5.0]
+  def change
+    create_table :units do |t|
+      t.string :name
+      t.integer :beds
+      t.string :floor
```

Comment:
I am thinking we may need to reify floor also.   A building has many floors, and a unit belongs to a floor.   It might be the case that most of the time there is only one floor and all units are on the first floor.  but for now we should add that structure I think